.changesets/feat_convertNonSecondTimeUnits.md

```diff
@@ -0,0 +1,27 @@
+### Automatic unit conversion for duration instruments with non-second units
+
+The Router now automatically converts duration measurements to match the configured unit for telemetry instruments.
+Previously, duration instruments always recorded values in seconds regardless of the configured `unit` field.
+With this enhancement, when you specify units like `"ms"` (milliseconds), `"us"` (microseconds), or `"ns"` (nanoseconds),
+the Router will automatically convert the measured duration to the appropriate scale.
+
+**Supported units:**
+- `"s"` - seconds (default)
+- `"ms"` - milliseconds
+- `"us"` - microseconds
+- `"ns"` - nanoseconds
+
+Important Note: This should only be used in cases where you are required to integrate with an observability platform that does not properly translate from source timeunit into the necessary target timeunit (ie: seconds to milliseconds).  In all other cases, 
+customers should follow the OLTP convention indicating you "SHOULD" use seconds as the unit.
+
+**Example:**
+```yaml
+telemetry:
+  instrumentation:
+    instruments:
+      router:
+        http.server.request.duration:
+          unit: "ms"  # Values are now automatically converted to milliseconds
```

Comment:
This configuration is probably wrong. Because it's not a custom metric it's a built-in/otel metric. So it would work for a custom one but not for built-in/otel ones.